cloudferrylib/scheduler/scenario.py

```diff
@@ -57,8 +61,11 @@ def init_tasks(self, init):
     def load_scenario(self, path_scenario=None):
         if path_scenario is None:
             path_scenario = self.path_scenario
+        if not os.path.isfile(path_scenario):
+            raise NotFoundScenarioFileError(path_scenario)
         with open(path_scenario) as scenario_file:
             migrate = yaml.load(scenario_file)
+            ScenarioChecker.check_structure(migrate)
```

Comment:
You implemented that method in the same class and you do not need to call the method using a name of the class. Python allows you to call the method using `self`